src/Sulu/Bundle/MediaBundle/Tests/Unit/DependencyInjection/SuluMediaExtensionTest.php

```diff
@@ -9,7 +9,17 @@
  * with this source code in the file LICENSE.
  */
 
-namespace Sulu\Bundle\SnippetBundle\Tests\Unit\DependencyInjection;
+namespace Symfony\Component\Process;
```

Comment:
Can this not throw some redeclare error?

Would go with something like the following instead:

```
    public function __construct(ExecutableFinder $executableFinder = null)
    {
        $this->executableFinder = $executableFinder ?: new ExecutableFinder();
    }
```

to mock the ExecutableFinder.

Symfony uses this to Unit Test there Messenger Connections.
https://github.com/symfony/symfony/blob/65a988880d719f0de2329b39eb2e7c1b56d7532f/src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php#L74

Reply:
Well, it does not throw any errors in our test suite... The other way feels a bit weird, because the class is not really designed to be used with something else... But it is ok, I am going to change it.